dashi/poolqc/pooling_qc_sample.py

```diff
@@ -9,79 +9,159 @@
+                      html.Br(),
+                      html.P(children='Lane Number', style={'padding-left': '5px', 'font-weight': 'bold'}),
+                      html.Div(
+                          dcc.RadioItems(
+                              id='lane_select',
+                              labelStyle={'display': 'inline-block', 'padding-left': '30px', }
+                          )
+                      ),
+                      html.Br(),
+                      html.P(children='Threshold Value for Index Clusters',
+                             style={'padding-left': '5px', 'font-weight': 'bold'}),
+                      html.Div(
+                          dcc.Input(
+                              id='index_threshold',
+                              placeholder='Press "Enter" when complete',
+                              debounce=True,
+                              type='number',
+                              value='0'
+                          )
+                      ),
+                      html.Br(),
+                      html.P(children='Sample QC Status',
+                             style={'padding-left': '5px', 'font-weight': 'bold'}),
+                      html.Div(
+                          dcc.Checklist(
+                              id='pass/fail',
+                              options=[
+                                  {'label': 'Passed Samples', 'value': 'Pass'},
+                                  {'label': 'Failed Samples', 'value': 'Fail'},
+                              ],
+                              value=['Pass', 'Fail'],
+                              labelStyle={'paddingLeft': 30}
+
+                          ),
+                      ),
+                      html.Div([
+                          html.P(children='Sample Type',
+                                 style={'padding-left': '5px', 'font-weight': 'bold'}),
+                          dcc.Dropdown(
+                              id='sample_type',
+                              options=[{'label': 'DNA: WG', 'value': 'WG'},
+                                       {'label': 'DNA: EX', 'value': 'EX'},
+                                       {'label': 'DNA: TS', 'value': 'TS'},
+                                       {'label': 'DNA: AS', 'value': 'AS'},
+                                       {'label': 'RNA: MR', 'value': 'MR'},
+                                       {'label': 'RNA: SM', 'value': 'SM'},
+                                       {'label': 'RNA: WT', 'value': 'WT'},
+                                       {'label': 'RNA: TR', 'value': 'TR'}],
+                              value=['WG', 'EX', 'TS', 'AS', 'MR', 'SM', 'WT', 'TR'],
+                              clearable=False,
+                              multi=True
+                          )])
+                  ])]
+              ),
+    sd.RaisedButton(id='filters',
+                    label='Filters'),
+    html.Div(children=[
+        # dcc.ConfirmDialog(
```

Comment:
Why is this commented out?

Reply:
@slazicoicr had requested in https://github.com/oicr-gsi/dashi/pull/38#discussion_r316218642

Reply:
So can you please remove it entirely or add a "TODO" with a link to a JIRA ticket?